app/models/locations/map.rb

```diff
@@ -0,0 +1,90 @@
+module Locations
+  class Map
+    include ActiveModel::Model
+
+    def initialize(params = {})
```

Comment:
@tpendragon how do you get it to populate the instance variable only on first access?  I was setting the instance variables with the setter functions on initialization because I didn't want to keep calling the HTTP request.  Also, someone told me that creating instance methods were better than `attr_accessor`... but I will do that to clean it up.